UnityProject/Assets/Scripts/Items/Tools/Extinguisher.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Extinguisher : MonoBehaviour
+{
+    public bool isOn = false;
```

Comment:
Can you make this private so it's more encapsulated? If another thing needs to check if it's on, you could expose it via a public getter method

like public bool IsOn => isOn;